biolink-model.yaml

```diff
@@ -6549,6 +6555,170 @@ slots:
+    range: integer
+
+  concept pair count:
+    is_a: association slot
+    description: >-
+      The number of instances in a dataset/cohort whose records contain
+      both the subject and object concept of an association.
+    range: integer
+    examples:
+      - value: 1731
+
+  expected count:
+    is_a: association slot
+    description: >-
+      The expected (calculated) number of instances in a dataset/cohort whose records contain both the subject and 
+      object concept of an association if the subject and object concepts are independent.
+
+  relative frequency subject:
+    is_a: association slot
+    description: >-
+      The frequency at which subject and object concepts co-occur in EHR
+      records within a dataset/cohort, relative to the frequency at which the subject
+      concept appears in these same records.
+    range: float
+    examples:
+      - value: 0.01840490798
+
+  relative frequency object:
+    is_a: association slot
+    description: >-
+      The frequency at which subject and object concepts co-occur in EHR
+      records within a dataset/cohort, relative to the frequency at which the object
+      concept appears in these same records.
+    range: string
+
+  relative frequency subject confidence interval:
+    is_a: association slot
+    description: >-
+      The 99% confidence interval for the relative_frequency_subject calculation
+      (i.e. the range of values within which the true value has a 99% chance of falling)
+    range: string
+
+  relative frequency object confidence interval:
+    is_a: association slot
+    description: >-
+      The 99% confidence interval for the relative_frequency_object calculation
+      (i.e. the range of values within which the true value has a 99% chance of falling)
+    range: string
+
+  adjusted p value:
+    is_a: association slot
+    description: >-
+      The adjusted p-value is the probability of obtaining test results
+      at least as extreme as the results actually observed, under the assumption that
+      the null hypothesis is correct, adjusted for multiple comparisons.  
+      P is always italicized and capitalized. The actual P value* should be expressed (P=. 04) 
+      rather than expressing a statement of inequality (P<. 05), unless P<.
+    range: float
+
+  bonferonni adjusted p value:
+    is_a: association slot
+    description: >-
+      The Bonferroni correction is an adjustment made to P values when several dependent or independent 
+      statistical tests are being performed simultaneously on a single data set. To perform a Bonferroni 
+      correction, divide the critical P value (α) by the number of comparisons being made.  P is always italicized and 
+      capitalized. The actual P value* should be expressed (P=. 04) rather than expressing a statement of inequality 
+      (P<. 05), unless P<.
+    range: float
+    examples:
+      - value: 0.018
+
+  supporting text:
+    is_a: association slot
```

Comment:
It occurred to me that many of the properties we are adding, like this one, will actually hang from a StudyResult object, not an Association - which I think means that technically they are not 'association slots', right?  I think that StudyResult object will always hang form Associations, so these slots are association_slots  in a indirect sense . . . . which may be enough justification to place them under 'association slot' int he biolink slot hierarchy.

Reply:
yep, we could have another high-level grouping slot "study result slot" as a sibling to association_slot perhaps; in particular if we need it for technical reasons (grouping all these together).  Holding off for now.

Reply:
Yeah, I would hold off.  I think many of these properties could hang directly from an Association in some cases, or be used within a Study Result in other cases.  Fine for no I think to not worry about this nuance, and just call them all Association slots (even if within an Association structure they are nested in some object that hangs from the Association directly)